lib/galaxy/util/commands.py

```diff
@@ -55,7 +55,7 @@ def redirect_aware_commmunicate(p, sys=_sys):
     return out, err
 
 
-def shell(cmds: Union[List[str], str], env: Optional[Dict[str, str]] = None, **kwds: Dict[str, Any]) -> int:
+def shell(cmds: Union[List[str], str], env: Optional[Dict[str, str]] = None, **kwds: str) -> int:
```

Comment:
For `**kwargs` the type annotation (pre Python 3.11) should be the type of the *values* (see e.g. https://peps.python.org/pep-0692/ ), which can be any in this case.

```suggestion
def shell(cmds: Union[List[str], str], env: Optional[Dict[str, str]] = None, **kwds: Any) -> int:
```

Reply:
Thanks!